source/releasenotes/2024-05-02-Synthetics-Check-Limits.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "New Relic Synthetics: New monthly Check limits Starting May 1st, 2024"
+published_date: "2024-05-02"
+categories: [action-required]
+---
+**New Relic Synthetics: New monthly Check limits**
+
+Beginning on May 1st, 2024, Pantheon will begin limiting the number of Synthetics checks for each Pantheon account.
+* Each Pantheon account will have the option to run up to 10,000 New Relic Synthetic checks per month.
+* Pantheon provides a monthly shared number of synthetic checks for all Pantheon customers each month. To ensure that all Pantheon users can use Synthetics, we ask customers to reduce their checks to <10,000 per month per Pantheon account.
+* Starting May 1st, 2024, Pantheon will take action and reduce the number of checks in Pantheon accounts exceeding the 10,000 checks per month limit.
+* Pantheon will not delete synthetic monitors or historical data but will reduce the frequency and the number of locations the monitor runs from.
+* If you have any questions, please feel free to contact [Pantheon Support , https://pantheon.io/support].
```

Comment:
Would recommend this not be a bullet line - it can be its own line. 

This link formatting is not following markdown standards

Reply:
This link was removed based on a previous comment.